src/bin/ngm-cli.ts

```diff
@@ -104,10 +105,10 @@ if (cli.input.length === 0) {
 }
 
 // project flag is mandatory for now
-if (!cli.flags.project) {
-  console.error('Please provide path to your projects source folder, `-p DIR` ');
-  process.exit(1);
-}
+// if (!cli.flags.project) {
```

Comment:
You will have to add another check

Reply:
It's there https://github.com/valor-software/npm-submodules/pull/17/files#diff-c3ca6689a5afcaf7010b439a1d03bdc3R169